From 69f428ef0c1d3cec10dc7c52907aa7da2d0a4386 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 6 Jun 2016 13:49:22 +0800
Subject: [PATCH 1091/5242] MLK-12886-1 ARM: dts: imx7d-sdb: the assert gpio
 for lcdif should be active low

commit  2e8fa2ea02f5cb76564c22d5b1fd74ffaaf22516 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to the 7d sdb schematic, only when the 'LCD_PWR_EN' is
low voltage, the 'LCD_3V3' can has the 3.3V voltage. And 'LCD_3V3'
is used to provide 3.3V power for lcd peripherals.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 86a8bf7..01d65b9 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -581,6 +581,7 @@
 &lcdif {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_lcdif>;
+	pinctrl-assert-gpios = <&extended_io 7 GPIO_ACTIVE_LOW>;
 	status = "okay";
 
 	port {
-- 
1.7.9.5

